### ISSUE TITLE
Estimación de horas para nuevos desarrollos para Del Viento

### ISSUE THREAD

**Issue by LeandroBraier** [LeandroBraier/DELVIENTO#3]
@mkreyness y @valentinsabbath les pido aquí lo siguiente:

. Calcular cuantas horas de cada uno les llevaría incorporar la funcionalidad de un segundo reel.

. Calcular horas de cada uno para desarrollo del sistema que funcione por internet sin lenovos.

. Cuantas horas de cada uno les llevaría sumar animaciones a los gustos.

**Comment by mkreyness** [comment]
Empecemos por lo mas conocido: Internet "sin lenovos":
Lo pienso en voz alta para que me vaya quedando como lista de tareas:
- Internet sin lenovos implica setear un server en Amazon, y lo mas logico seria apuntarle algun subdominio de delviento.com, algo asi como siguit.delviento.com, o capaz mejor aun delviento.siguit.com
- Setear un server nuevo siempre lleva unas horitas, ya que vienen practicamente pelados.
- Y despues hay que mudar el software, las imagenes Y los datos de benteveo.tv ahi.
- Tambien hay que hacer algunas adaptaciones para que todo el manejo de gustos por local funcione y genere videos independientes para cada local.
- Hay que hacer una nueva "tarjeta base" de raspi que acceda al dominio que elijamos, y sincronice. Esto mas o menos ya esta encaminado con Valen.
- ponele mias entonces esto de la mudanza unas 25 horas.

Lo de la animacion:
- como te decia antes lo mas engorroso es la carga de los 150 png por gusto, despues el armado en si no deberia ser gran problema, tengo que hacer memoria de como estaba armado todo esto e intervenirlo. Ponele 10 horas mias.

Finalmente lo del segundo reel: 
- esto no lo tengo muy claro sobre todo en cuanto al CONTENIDO de ese segundo reel, cuan VARIABLE seria ? Depende de cada local, tiene algo que ver con los gustos, o no ? Porque si es algo totalmente independiente del primero (que es el que se genera por sistema), y siendo que ahora se hace via Infobeamer, va a ser mejor que lo maneje enteramente Valen.
- Yo diria de reenfocar "lo que sucede en la raspi" como un proceso de Infobeamer en el cual a veces se muestra la parte de gustos que corresponde a esa raspi, y a veces se muestra el "2do reel que corresponde a esa raspi" (la parte izquierda, central o derecha del "wall"), y asi lo puede manejar 100% Valen.

**Comment by LeandroBraier** [comment]
Excelente MK! Muchísimas gracias!

Espero la respuesta de Valen para cerrar las horas de ambos.

**Comment by ghost** [comment]
@mkreyness algunas preguntas:
En cuanto a las "memorias plug and play" me falta muy poco. Pero estando en la nube, la descarga va a ser por wget?
El reel va a precisar de un schedule por cron que va a generar el motor de DelViento?

**Comment by mkreyness** [comment]
No, rsync, siempre usemos rsync para evitar bajar cosas que no cambiaron.
No tenemos previsto un schedule real por hora del dia, es mas bien una rutina que se repite siempre, pero DENTRO de la rutina hay 2 etapas: Reel1 / Reel2.
El asunto es que como el Reel1 de cada puesto va desfasado (para que no cambien todos juntos, sino que la gente pueda ir mirando uno, despues otro, etc., y el Reel2 va coordinado, hay que pensar bien como armarlo en el infobeamer. El jueves lo charlamos.
